README.md

```diff
@@ -25,9 +25,9 @@
 - [❓ FAQ (OPTIONAL)](#faq)
```

Comment:
Remove this section because it is not in use within the README file. It is important to avoid having broken links within the README.md file. Broken links are not good for user experience.

Reply:
- [ ] Great job @lucy-sees I will suggest you make this comment sound a bit more polite. Currently, it sounds more like a command than a suggestion so kindly work on this :+1: 
- [ ] For example: This is a polite suggestion: `I think it would be better to use semantic tags`
- This reads as an order: `Use semantic tags`

Reply:
I have made the change. Thank you for pointing it out. 😊